docs/api/tray.md

```diff
@@ -79,7 +79,15 @@ app.whenReady().then(() => {
 ### `new Tray(image, [guid])`
 
 * `image` ([NativeImage](native-image.md) | string)
-* `guid` string (optional) _Windows_ - Assigns a GUID to the tray icon. If the executable is signed and the signature contains an organization in the subject line then the GUID is permanently associated with that signature. OS level settings like the position of the tray icon in the system tray will persist even if the path to the executable changes. If the executable is not code-signed then the GUID is permanently associated with the path to the executable. Changing the path to the executable will break the creation of the tray icon and a new GUID must be used. However, it is highly recommended to use the GUID parameter only in conjunction with code-signed executable. If an App defines multiple tray icons then each icon must use a separate GUID.
+* `guid` string (optional) _Windows_ _macOS_ - A unique string used to identify the tray icon.
```

Comment:
Since up until now this has only been a positional parameter, it seems like we could rename the param without introducing a breaking change. I would suggest calling this `identifier`, because a GUID is a specific type of string, and it sounds like only the Windows API specifically requires that the identifier be a GUID.

Reply:
Agreed here. On first read, I don't think it was clear that the macOS string can just be a unique string rather than a GUID based on the name.